.atomist/handlers/event/Deployed.ts

```diff
@@ -1,35 +1,47 @@
-import { HandleEvent, Message, Plan } from '@atomist/rug/operations/Handlers'
+import { HandleEvent, ChannelAddress, DirectedMessage, LifecycleMessage, MessageMimeTypes, ResponseMessage, Plan } from '@atomist/rug/operations/Handlers'
 import { GraphNode, Match, PathExpression } from '@atomist/rug/tree/PathExpression'
 import { EventHandler, Tags } from '@atomist/rug/operations/Decorators'
 import * as query from '@atomist/rugs/util/tree/QueryByExample'
-import { Pod } from "@atomist/cortex/stub/Pod"
-import { Container } from "@atomist/cortex/stub/Container"
-import { Tag } from "@atomist/cortex/stub/Tag"
-import { Commit } from "@atomist/cortex/stub/Commit"
-import { Repo } from "@atomist/cortex/stub/Repo"
-import { ChatChannel } from "@atomist/cortex/stub/ChatChannel"
+import { K8Pod } from "@atomist/cortex/K8Pod"
+import { DockerImage } from "@atomist/cortex/DockerImage"
+import { Tag } from "@atomist/cortex/Tag"
+import { Commit } from "@atomist/cortex/Commit"
+import { Repo } from "@atomist/cortex/Repo"
+import { ChatChannel } from "@atomist/cortex/ChatChannel"
 
 @EventHandler("pod-deployed", "Handle Kubernetes Pod deployment events", 
-     `/Pod()
+     `/K8Pod()
         [@state='Started']
-        [/uses::Container()
-            [/isTagged::Tag()
-                [/isTagged::Commit()
-                    [/on::Repo()/channel::ChatChannel()]]]]`
+        [/images::DockerImage()
+            [/tag::Tag()
+                [/commit::Commit()
+                    [/builds::Build()
+                        [/commit::Commit()]
```

Comment:
Actually I don't think you need this commit either as you are taking it from the one above when constructing the correlation-id (although you've deleted the code in this PR) and thinking more on it - that should be the correct sha to correlate with. I think you can remove it from all of them.